test/BurnableMintableCappedERC20.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+const {
+  Contract,
+  utils: { defaultAbiCoder, splitSignature, arrayify },
+} = require('ethers');
+const { deployContract, MockProvider, solidity } = require('ethereum-waffle');
+const { get } = require("lodash/fp");
+const chai = require('chai');
+chai.use(solidity);
+const { expect } = chai;
+
+const CHAIN_ID = 1;
+const ADDRESS_ZERO = '0x0000000000000000000000000000000000000000';
+const ROLE_OWNER = 1;
+
+const BurnableMintableCappedERC20 = require('../build/BurnableMintableCappedERC20.json');
+
+const AxelarGatewayProxyMultisig = require("../build/AxelarGatewayProxyMultisig.json");
+const AxelarGatewayMultisig = require("../build/AxelarGatewayMultisig.json");
+const {getRandomID, getSignedMultisigExecuteInput} = require("./utils");
+
+describe('BurnableMintableCappedERC20', () => {
+  const wallets = new MockProvider().getWallets();
+  const owners = wallets.slice(0, 3);
+  const operators = wallets.slice(3, 6);
+  const admins = wallets.slice(6, 9);
+  const threshold = 2;
+
+  const ownerWallet = owners[0]
+  const userWallet = admins[0]
+
+  const tokenName = 'Test Token';
+  const tokenSymbol = 'TEST';
+  let gateway;
```

Comment:
Do you absolutely need a gateway to test `BurnableMintableCappedERC20.js?

Reply:
Moved the `burnFrom` back into `MintableCappedERC20` so we don't need the gateway to deploy it